Assets/Scripts/ProjController.cs

```diff
@@ -0,0 +1,74 @@
+// proj = projectile
```

Comment:
To clarify why I requested `proj` be changed to `projectile`: I think it's clear in context what `proj` is short for, but it might be unclear what `proj` means when it's not close to the word projectile.